code/modules/multiz/level_data.dm

```diff
@@ -525,13 +523,19 @@ INITIALIZE_IMMEDIATE(/obj/abstract/level_data_spawner)
 
 /datum/level_data/mining_level/asteroid
 	base_turf = /turf/simulated/floor/asteroid
+	level_generators = list(
+		/datum/random_map/automata/cave_system,
+		/datum/random_map/noise/ore
+	)
 
 /datum/level_data/mining_level/generate_level()
 	//#FIXME: This config option is very ambiguous. Most RNG stuff doesn't care about it. Might be worth removing?
```

Comment:
When talking about map generation, do you mean solely like noise maps used to generate terrain features, or like map stitching randomly picked away sites/ruins? 
Because, I assumed the purpose of this var was to prevent random noise terrain gen from happening, seeing as it was preventing that on mining turfs?

Reply:
But, won't that means we'll skip completely testing level gen during unit tests though? I'm seeing defines that force change the var to skip map gen as long as UNIT_TEST is defined.
Like, is it really worth the shortened testing time if we skip testing something kinda important like that? Picking and placing ruins, generating exoplanet terrain, generating mining data, etc..

Reply:
Renamed the option and made it apply to all the obvious map gen stuff I could find.